scripts/travis_build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
+# Doing jobs according to COMPONENT's value to minimize build times
+
+if [ "$JOB" == "BUILD" ]; then
+    bundle exec jekyll build
```

Comment:
We don't need this line.

Simply execute the img_checker and/or build node in parallel.